### DISCUSSION TITLE
Plactica HTML

### DISCUSSION THREAD

**Discussion by Armandoamtz** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#232] [Show and tell]
Hola compañeros, les comparto mi práctica de la pasteleria, espero les guste y les de inspiración para su proyecto

https://paste-launchx.000webhostapp.com/

**Comment by JuanSalvador73** [comment]
Me gustó mucho, buen trabajo.

**Comment by enriqueorea** [comment]
Muy buena, pero recuerda que la navbar es una parte fundamental de tu página

**Comment by hackit92** [comment]
Creo que en general muy bien, pero como comenta Enrique, la falta de navegación en el sitio no ayuda mucho, es importante seguir cuidando la experiencia del usuario basándonos en lo aprendido con la práctica de abogabot

**Comment by Armandoamtz** [comment]
Tienen razón, había olvidado la experiencia de usuario, muchas gracias por sus consejos

**Comment by angelusnovuz** [comment]
Donde encontraste las imágenes de los pasteles?

**Comment by OscarPedroza** [comment]
Hola compañero, buen diseño y muy buen trabajo.
